content/en/docs/languages/java/getting-started.md

```diff
@@ -16,7 +16,7 @@ console.
 
 Ensure that you have the following installed locally:
 
-- Java 17 JDK
+- Java 17+ JDK
```

Comment:
Unrelated to this PR, but someone reading this may infer that we only support Java 17+. This is not the case. We support java 8+ but this specific example requires java 17+ because of spring. You also need java 17+ if you're going to build `opentelemetry-java` from source, but it is not needed to consume the artifacts.

Reply:
Good point. Is this documented somewhere else? I think that we want to keep the "story" simple in the Getting Started page, but maybe link to somewhere else for the "entire story" :).

An alternative would be to add a footnote here. WDYT?

Reply:
We document it [here](https://github.com/open-telemetry/opentelemetry-java?tab=readme-ov-file#requirements):

> Unless otherwise noted, all published artifacts support Java 8 or higher.

The JDK version compatibility is a pretty important point in the java ecosystem, so while the entire story may be documented elsewhere, I think any spot which mentions version requirement conflicting with java 8+ is prone to being misinterpreted. A footnote is probably best here.

Perhaps we introduce a brief section on the [manual instrumentation](https://opentelemetry.io/docs/languages/java/instrumentation/) page describing the java 8+ requirement, and link to it from this getting started page?

Reply:
> The JDK version compatibility is a pretty important point in the java ecosystem, so while the entire story may be documented elsewhere, I think any spot which mentions version requirement conflicting with java 8+ is prone to being misinterpreted. A footnote is probably best here.

👍

Reply:
@jack-berg @trask - I've updated the prose. PTAL and let me know what you think. For now I'm trying to confine this change to the [**Getting started** page](https://deploy-preview-3877--opentelemetry.netlify.app/docs/languages/java/getting-started/#prerequisites):

> <img width="450" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/c853ff6a-1721-4aa3-adb7-0760cf569143">